biorbd_optim/problem_type.py

```diff
@@ -166,5 +166,5 @@ def get_q_from_V(ocp, V, num_phase=None):
             x, _, _, _ = ProblemType.get_data_from_V(ocp, V, num_phase)
 
         else:
-            raise RuntimeError (ocp.nlp[0]["problem_type"] + " is not implemented for this type of OCP")
+            raise RuntimeError(ocp.nlp[0]["problem_type"] + " is not implemented for this type of OCP")
```

Comment:
Use f string 
`f"{ocp.nlp[0]["problem_type"]} is not implemented for this type of OCP"`

Reply:
done in some files...